src/main/_docs/write-che-ide-plugins/plugins-agents.md

```diff
@@ -0,0 +1,168 @@
+---
+tags: [ "eclipse" , "che" ]
+title: Creating Custom Agents
+excerpt: ""
+layout: docs
+permalink: /:categories/languageserver/
```

Comment:
Why is this called `languageserver`? Also for links to work on right nav `languageserver` needs to be `agents` or filename needs to change to plugins-languageserver.md .